tasks/resolve_reference.json

```diff
@@ -27,6 +27,18 @@
     "calculate_transport": {
       "description" : "TODO Defaults to true",
       "type": "Optional[Boolean]"
+    },
+    "filter_older_than_days": {
+      "description" : "Filter out computers that have a filter_property time older than this many days.",
+      "type": "Optional[Integer]"
+    },
+    "filter_older_attribute": {
+      "description" : "LDAP attribute to use for filtering out old computers. Some good choices are pwdLastSet, lastLogonTimestamp",
+      "type": "Optional[String]"
+    },
+    "filter_dns_hostnames": {
```

Comment:
I'd probable call this `ignore_dns_hostnames` as it's quite specific

Reply:
I was just going with the `filter_` prefix because it was consistent with the other filtering features i had added. I can change the name if you like.